docs/plugins.md

```diff
@@ -0,0 +1,351 @@
+# Plugin Handbook
+
+## Table of Contents
+
+1. [Introduction](#introduction)
+   1. [What is a Lighthouse Plugin?](#what-is-a-lighthouse-plugin)
+   1. [Comparing a Plugin vs. Custom Config](#comparing-a-plugin-vs-custom-config)
+   1. [Getting Started](#getting-started)
+1. [API](#api)
+   1. [Plugin Config](#plugin-config)
+   1. [Plugin Audits](#plugin-audits)
+1. [Best Practices](#best-practices)
+   1. [Naming](#naming)
+   1. [Scoring](#scoring)
+   1. [Common Mistakes](#common-mistakes)
+1. [Examples](#examples)
+
+## Introduction
+
+If you're new to Lighthouse development, start by reading up on the overall [architecture](./architecture.md) and how [configuration](./configuration.md) works before continuing.
+
+### What is a Lighthouse Plugin?
+
+Lighthouse plugins are a way to extend the functionality of Lighthouse with insight from domain experts (that's you!) and easily share this extra functionality with other Lighthouse users. At its core, a plugin is a node module that implements a set of checks that will get run by Lighthouse and added to the report as a new category.
+
+![picture of Lighthouse plugin results in the HTML report](https://user-images.githubusercontent.com/2301202/57053947-24fc9b80-6c57-11e9-8853-df3a174fcf2c.png)
+
+### Comparing a Plugin vs. Custom Config
+
+Plugins are easily shared and have a stable API that won't change between minor version bumps but are also more limited in scope than a [custom Lighthouse configuration](./configuration.md). Before getting started with plugins, think about your current needs, and consult the table below to decide which is best for you.
+
+| Capability                                   | Plugin | Custom Config |
+| -------------------------------------------- | ------ | ------------- |
+| Include your own custom audits               | ✅     | ✅            |
+| Add a custom category                        | ✅     | ✅            |
+| Easily shareable and extensible              | ✅     | ❌            |
```

Comment:
actually maybe it is good to name drop npm here just to get people thinking about the possibilities :)

Reply:
```suggestion
| Easily shareable and extensible on NPM       | ✅     | ❌            |
```